helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -13,11 +13,10 @@
       <field name="name">Ticket Assignment</field>
       <field name="model_id" ref="model_helpdesk_ticket"/>
       <field name="email_from">${object.company_id.partner_id.email}</field>
-      <field name="email_cc">${not object.partner_id and object.partner_email or ''|safe},</field>
       <field name="subject">${object.company_id.name} Ticket Assignment (Ref ${object.number or 'n/a' })</field>
-      <field name="partner_to">${object.partner_id.id}</field>
+      <field name="email_to">${object.user_id.partner_id.email}</field>
```

Comment:
Change this to: `<field name="partner_to">${object.user_id.partner_id.id}</field>`

Reply:
The partner is the only way to send "a assigned to" mail (there is no separate mail field)

Reply:
@DarioLodeiros Thank you, this appears to have stopped sending the customer an 'assigned to' email when a ticket is created from an inbound email. The odoo user doesn't get any notification though, nothing in 'discuss' (the users have 'handle notification in odoo' set). Is this the same for others?